apache/docker-entrypoint.sh

```diff
@@ -75,9 +75,11 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 	if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] === 'https') {
 		$_SERVER['HTTPS'] = 'on';
 	}
-	
+	foreach (glob("/etc/wordpress/*.php") as $filename)
```

Comment:
This is awesome, however I'd rather merge in additional configs in `docker-entrypoint.sh`. Doing a `glob` for each request seems a bit expensive, doesn't it?